subjects/make-your-game/README.md

```diff
@@ -0,0 +1,114 @@
+## make-your-game
+
+As times passes and technology evolves, the human brain has been requesting
+more and more stimuli to keep the brain occupied. Boredom happens whenever
+people don't receive enough of these stimuli. And you are not receiving enough
+of it, so you decide to do **your own game**. And you also decide to just use
+JavaScript as a challenge.
+
+Your game will be for single player only and you will have to create your
+own engine and tools so that the game works. And as you try to search all sort
+of information you start to remember all those [60fps memes](https://pics.me.me/60-fps-59-fps-35518800.png).
+
+### Objectives
+
+Here are some of the features you want to implement on your game:
+
+- Run the game at least at **60fps** at all time
+- You must not have frame drops!
+- You will use **RequestAnimationFrame**
```

Comment:
We should say:
- proper use of `RequestAnimationFrame`

It's not about using it but using it correctly